File: inspect_form.py
import http.client
import json

conn = http.client.HTTPConnection("localhost", 5000)
conn.request("GET", "/api/forms/2")
response = conn.getresponse()
data = response.read().decode()
try:
    obj = json.loads(data)
    print(json.dumps(obj, indent=2))
except Exception as e:
    print(f"Error parsing JSON: {e}")
    print(f"Raw data: {data}")
conn.close()

File: inspect_form.js
const http = require('http');

http.get('http://localhost:5000/api/forms/2', (res) => {
  let data = '';
  res.on('data', (chunk) => { data += chunk; });
  res.on('end', () => {
    try {
      console.log(JSON.stringify(JSON.parse(data), null, 2));
    } catch (e) {
      console.log('Error:', e.message);
      console.log('Raw:', data);
    }
  });
}).on('error', (err) => {
  console.log('Error:', err.message);
});
